php/nette/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "require": {
+        "nette/application": "^3.0",
```

Comment:
Could you use `~3.0.0`  ? We need to stick to **3.0**, the `^` notation could change the major

Reply:
sure, I will change